wiki/Twitter/vi.md

```diff
@@ -0,0 +1,24 @@
+# Twitter
+
+osu! có một vài tài khoản Twitter khác nhau để phục vụ những mục đích khác nhau. Phần lớn tài khoản này không hoạt động nhiều nhưng chúng sẽ giúp bạn cập nhật với tin tức về osu!.
```

Comment:
```suggestion
osu! có một vài tài khoản Twitter khác nhau, mỗi cái phục vụ cho mục đích cụ thể. Phần lớn trong số đó ít khi hoạt động nhưng chúng sẽ giúp bạn nắm bắt được thông tin ở ngoài hệ sinh thái osu!.
```
`osu! ecosystem` here means like `Apple ecosystem`, `Android ecosystem`, ...
It helps you "informed", or "up-to-date" about news, ... outside of osu! web, forum, in-game (ecosystem).
You made it means `help you update news about osu!` only.